### PR TITLE
Update `CHANGELOG.md` for the 0.8.0 release

### PR DESCRIPTION
**Pull request by MaxDesiatov**
We haven't had a release for a while, time to make all these new features available for a wider audience!

### CODE REVIEW

**Comment by carson-katri** [comment]
Do you think #435 could be finished first so that this release is compatible with the latest SwiftUI betas?

**Comment by MaxDesiatov** [comment]
Sure, would you have time to address feedback on that PR?

**Comment by carson-katri** [comment]
Yes, starting on that now

**Comment by MaxDesiatov** [comment]
Thanks so much! 🙏 No need to rush, I can postpone a new release for a week or two if you need more time.

**Comment by MaxDesiatov** [comment]
@TokamakUI/tokamak-team would anyone mind reviewing this? Or maybe any objections against releasing this tomorrow?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,72 @@
+# 0.8.0 (17 August 2021)
+
+This release adds support for more SwiftUI types and modifiers, and fixes bugs. Including, but not
+limited to:
+
+- `Toolbar` type and `toolbar` modifier
+- `ProgressView` type
+- `Animation` and related types and modifiers
+- `opacity`, `scaleEffect`, `aspectRatio`, `controlSize`, and `controlProminence` modifiers
```
Comment (carson-katri) [inline CHANGELOG.md:9]:
```suggestion
- `opacity`, `scaleEffect`, `aspectRatio`, and `controlSize` modifiers
```
`controlProminence` was removed in #435 and replace with `BorderedProminentButtonStyle`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,72 @@
...
+- `Material` and `Gradient` types
+- `Primary`/`Secondary`/`Tertiary`/`QuaternaryContentStyle` types
```
Comment (carson-katri) [inline CHANGELOG.md:11]:
```suggestion
- `HierarchicalShapeStyle` (`.primary`/`.secondary`/`.tertiary`/`.quaternary`) type
```
These styles were replaced with a single `HierarchicalShapeStyle` type in #435

**Review by carson-katri — APPROVED** [review]

**Pull request merged**